docs/network-edge/domains-and-tcp-addresses.mdx

```diff
@@ -129,17 +129,16 @@ IPs](#ip-addresses).
 
 ### GSLB support
 
-Unlike [Domains](#domains), TCP Addresses are not region-independent. They do
-not support [global server load balancing](/network-edge/gslb). When you create
-a TCP Address, you provision it in a region of your choice. Connections to that
-address are received exclusively in that region.
+Similar to [Domains](#domains), TCP Addresses are region-independent. They
+support [global server load balancing](/network-edge/gslb). TCP addresses
```

Comment:
TCP addresses do not yet support global server load balancing. for that to work, we need BGP-level route control, which we don't do yet.

Reply:
thanks! updated!